DEF.java

```diff
@@ -0,0 +1,23 @@
+public class DEF {

```

Comment:
Create enum with values

Reply:
instead of DEF class? or inside of it?

Reply:
Instead, no need for a class only for const values